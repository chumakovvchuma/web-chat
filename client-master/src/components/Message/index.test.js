import { h } from 'preact';
import render from 'preact-render-to-string';

import Message from '.';

describe('<Message />', () => {
  it('matches snapshot', () => {
    const props = {
      content: [], // message content
      type: 'client-TEST',
    };

    const tree = render(<Message type={props.type} content={props.content} />);
    expect(tree).toMatchSnapshot();
  });
});
