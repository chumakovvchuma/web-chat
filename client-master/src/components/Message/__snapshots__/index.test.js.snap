// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Message /> matches snapshot 1`] = `"<li class=\\"Message__box--undefined\\"><ul class=\\"Message__client--undefined\\"></ul></li>"`;
